mypy.ini

```diff
@@ -2,4 +2,4 @@
 files = decoy,tests
 strict = True
 show_error_codes = True
-plugins = decoy.mypy
+plugins = decoy/mypy/plugin.py
```

Comment:
This change required to unbreak mypy linting in VSCode due to the addition of some sort of path isolation added by microsoft/vscode-python#10670 that prevents the VSCode mypy from finding the locally installed `decoy` package (installed in editable mode by `poetry`)